src/data.jl

```diff
@@ -0,0 +1,41 @@
+function register_datadep()
+    register(
+        DataDep(
+            "impute-v0.1.0",
```

Comment:
@glennmoy  wrote some standards in DataZoo for how to version data.
I think it should be version from 1.0.0

Reply:
Probably not the DataDeps docs.
Since DataDeps doesn't handle versioning.
Putting them somewhere public would be good.

Maybe a blog post would be a good start